src/org/apache/commons/httpclient/HttpConnection.java

```diff
@@ -840,7 +840,7 @@ public void setTunnelRequested(boolean tunnelRequested) {
     /**
      * Indicates if the connection is completely transparent from end to end.
      *
-     * @return true if conncetion is not proxied or tunneled through a transparent
+     * @return true if connection is not proxied or tunneled through a transparent
```

Comment:
Please, revert the changes in these forked classes, they should have just essential/required changes for our use cases.

Reply:
Done